node/src/command.rs

```diff
@@ -83,14 +83,18 @@ fn load_spec(
     id: &str,
     para_id: ParaId,
 ) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
-    log::info!("KOM=> id-{}, para_id-{}", id, para_id);
+    log::info!("load_spec()=> id-{}, para_id-{}", id, para_id);
     Ok(match id {
         "dev" => Box::new(chain_spec::cs_main::development_config()),
         "local" => Box::new(chain_spec::cs_main::local_testnet_config()),
         "staking-dev" => Box::new(chain_spec::cs_staking::development_config()),
         "staking-local" => Box::new(chain_spec::cs_staking::local_staking_config()),
+        "eden" => Box::new(chain_spec::cs_eden::ChainSpec::from_json_bytes(
+            &include_bytes!("../res/eden.json")[..],
+        )?),
         "eden-dev" => Box::new(chain_spec::cs_eden::development_config(para_id)),
         "eden-local" => Box::new(chain_spec::cs_eden::local_config(para_id)),
+        "eden-valid" => Box::new(chain_spec::cs_eden::valid_config(para_id)),
```

Comment:
I am not clear what is the purpose of `eden-valid`?

Reply:
I guess it's the spec for westend. In that case eden-westend could be a better name. My question is what is the use case of eden-local vs eden-dev?

Reply:
Changed to "eden-util", for the purpose mentioned 
in description of the PR. I hope I am not over killing the code.

Reply:
If we plan on removing it at some point in the future, then this placeholder seems acceptable.